src/Common/src/Interop/Linux/procfs/Interop.ProcFsStat.cs

```diff
@@ -191,7 +191,7 @@ internal static bool TryReadStatFile(int pid, out ParsedStat result, ReusableTex
         internal static bool TryReadStatFile(int pid, int tid, out ParsedStat result, ReusableTextReader reusableReader)
         {
             bool b = TryParseStatFile(GetStatFilePathForThread(pid, tid), out result, reusableReader);
-            Debug.Assert(!b || result.pid == tid, "Expected thread ID from stat file to match supplied tid");
+            //Debug.Assert(!b || result.pid == tid, "Expected thread ID from stat file to match supplied tid");
```

Comment:
Why is this commented out?  If it's necessary, could you add a comment explaining and linking to whatever the associated issue # is?